lib/relay.js

```diff
@@ -144,6 +144,7 @@ function forwardHttpRequest(filterRules) {
       requestHeaders: req.headers,
       requestId: req.headers['snyk-request-id'],
       maskedToken: req.maskedToken,
+      hashToken: req.hashedToken,
```

Comment:
nitpick: how about to use the following name `hashedToken: req.hashedToken`?

Reply:
Yes I missed that. The auto selection had picked the function name. Will amend later today